frontend/src/metabase/components/ListItem/ListItem.jsx

```diff
@@ -30,9 +31,7 @@ const ListItem = ({
         </div>
         <div className={S.itemBody}>
           <div className={S.itemTitle}>
-            <ListItemName tooltip={name} tooltipMaxWidth="100%">
-              <h3>{name}</h3>
-            </ListItemName>
+            <Ellipsified tooltip={name}>{name}</Ellipsified>
```

Comment:
I killed this styled component because it just made it all harder to reason about